forked.generated.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+
+# Parachin chains configurataion files
+FORKED_CONFIG_FILE="config.parachain.agung.forked.yml"
+RPC_ENDPOINT="https://rpcpc1-qa.agung.peaq.network"
+DOCKER_COMPOSE_FOLDER="yoyo"
+FORK_FOLDER="/home/jaypan/Work/peaq/fork-test/fork-binary/peaq-dev-v06042023"
```

Comment:
Here again the local path - does it work on every computer?

Reply:
Yep, change that to ask the user to pass it